notebooks/responsible_ai/explainable_ai/solutions/xai_text_with_LIT.ipynb

```diff
@@ -0,0 +1,2770 @@
+{
```

Comment:
<p>Under "Customize the model for interpretability methods" add a list of the techniques you are covering and maybe link them since this is a long notebook.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/527/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>The list is already written a few lines above, so added the link there!</p>